pkgs/by-name/al/algia/package.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "algia";
+  version = "0.0.74";
+
+  src = fetchFromGitHub {
+    owner = "mattn";
+    repo = "algia";
+    rev = "v${version}";
+    sha256 = "sha256-t6XDw40FTa7QkZmOkgAufWV1aFjQrLWmycp+zcVYQWs=";
+  };
+
+  vendorHash = "sha256-fko9WC/Rh5fmoypqBuFKiuIuIJYMbKV+1uQKf5tFil0=";
+
+  meta = with lib; {
+    description = "A cli application for nostr";
```

Comment:
```suggestion
    description = "CLI application for nostr";
```

Reply:
I wrote the description quoted by: https://github.com/mattn/algia#:~:text=About-,A%20cli%20application%20for%20nostr,-Topics

Should I rewrite the description to suggested one?

Reply:
see https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

Reply:
OK, I understood!!
resolved: fdb1ad0d30e8c69339480cbc1f9848b73f9fb5d4